docs/modules/ROOT/pages/object-storage/create.adoc

```diff
@@ -29,6 +29,28 @@ spec:
 <4> Secret where the connection details are provisioned. This secret shouldn't exist at this time.
 +
 IMPORTANT: Object Storage Bucket names must be unique across providers. Make sure to choose a name with prefixes and other guards, to prevent any issue during the creation of the bucket. No error message is returned if the bucket cannot be created.
+== Create a Bucket in another cloud provider
+
+By default the bucket is created on the same cloud provider where the cluster is running on.
+For instance, if you are on an Exoscale cluster, to create an ObjectBucket on cloudscale.ch add the following YAML snippet to the `spec` of an ObjectBucket:
+
+[source,yaml]
+----
+spec:
+  compositionRef:
+    name: cloudscale.objectbuckets.appcat.vshn.io <1>
+  parameters:
+    region: rma <2>
+----
+<1> The reference name of the backing cloud provider
+<2> Change the region identifier in accordance for the resp. cloud provider
```

Comment:
Where can I find a list of region identifiers?

Reply:
Ah, that would be here: https://docs.appuio.cloud/appcat/object-storage/zones.html. Please link this page here and make it available in the navigation, it's missing there.

Reply:
I added the link in the navigation page. Also linked the zones in this guide as well.